documentation/book/con-custom-resources-example.adoc

```diff
@@ -64,16 +64,17 @@ kind: KafkaTopic <1>
 metadata:
   name: my-topic
   labels:
-    strimzi.io/cluster: my-cluster
-spec: <2>
+    strimzi.io/cluster: my-cluster <2>
+spec: <3>
   partitions: 1
   replicas: 1
   config:
     retention.ms: 7200000
     segment.bytes: 1073741824
 ----
 <1> The `kind` and `apiVersion` identify the CRD of which the custom resource is an instance.
-<2> The spec shows the number of partitions and replicas for the topic as well as configuration for the retention period for a message to remain in the topic and the segment file size for the log.
+<2> A `kafkaTopic` resource includes a label that defines the name of the Kafka cluster (derived from the name of the `Kafka` resource) to which it belongs. xref:con-topic-operator-cluster-label-deploying[The name is used by the Topic Operator to identify the Kafka cluster when creating the topic].
```

Comment:
Should start just with "This label defines the name ....".
At least this is the way I think about callouts because you have <2> on the side of which it describes.

Reply:
I've shortened the description to 'A label that defines...' I thought we needed to be clear about the label in relation to a `KafkaTopic` resources, but reading back it's fine to drop that intro.